node/node.d.ts

```diff
@@ -885,22 +885,144 @@ declare module "os" {
         internal: boolean;
     }
 
-    export function tmpdir(): string;
-    export function homedir(): string;
-    export function endianness(): "BE" | "LE";
     export function hostname(): string;
-    export function type(): string;
-    export function platform(): string;
-    export function arch(): string;
-    export function release(): string;
-    export function uptime(): number;
     export function loadavg(): number[];
-    export function totalmem(): number;
+    export function uptime(): number;
     export function freemem(): number;
+    export function totalmem(): number;
     export function cpus(): CpuInfo[];
+    export function type(): string;
+    export function release(): string;
     export function networkInterfaces(): { [index: string]: NetworkInterfaceInfo[] };
-    export function userInfo(options?: {encoding: string}): { username: string, uid: number, gid: number, shell: any, homedir: string }
+    export function homedir(): string;
+    export function userInfo(options?: { encoding: string }): { username: string, uid: number, gid: number, shell: any, homedir: string }
+    export function constants(): {
```

Comment:
https://nodejs.org/api/os.html#os_os_constants
I think this property is not function.